packages/sirius-web/frontend/sirius-web/src/onboarding/OnboardArea.tsx

```diff
@@ -69,7 +72,31 @@ const INITIAL_STATE = {
   representations: [],
 };
 
+const useOnboardAreaStyles = makeStyles((theme) => ({
+  grid: {
+    paddingTop: theme.spacing(6),
+    paddingBottom: theme.spacing(1),
+    paddingRight: theme.spacing(1),
+    paddingLeft: theme.spacing(1),
+    marginTop: theme.spacing(2),
+    marginLeft: theme.spacing(2),
+    marginRight: theme.spacing(2),
```

Comment:
Do not rely on margin, use padding to specify the layout of an element and instead of margin, use some specific layout on the parent element. It simplify the reuse of the element.